src/endpoints/getPlayerRanking.ts

```diff
@@ -1,18 +1,27 @@
+import querystring from 'querystring';
 import { PlayerRanking } from '../models/PlayerRanking'
 import { HLTVConfig } from '../config'
 import { fetchPage, toArray } from '../utils/mappers'
 
 export const getPlayerRanking = (config: HLTVConfig) => async ({
   startDate,
-  endDate
+  endDate,
+  matchType,
+  rankingFilter
 }: {
   startDate: string
   endDate: string
+  matchType: string
+  rankingFilter: string
```

Comment:
Considering that Top 5, 10, 20, 30 and 50 are the only available options (and I tried changing the url to something else), it's better to have this property as an enum, just like MatchType.